preamble.tex

```diff
@@ -73,17 +74,17 @@
 %\renewcommand{\topmargin}{-20pt}
 
 \rfoot{Page \textbf{\thepage} of \textbf{\pageref{LastPage}}}
-\lfoot{\textit{Final rules as of \today}}
+\lfoot{\textit{Draft rules as of \today}}
```

Comment:
@dbscoach if these are to be made final, should this change as well?

Reply:
Absolutely, my CTRL-F didn't find that...